src/components/KCodeBlock/KCodeBlock.vue

```diff
@@ -192,41 +192,57 @@
         v-if="isShowingFilteredCode"
         class="k-filtered-code-block"
         data-testid="k-code-block-filtered-code-block"
-      ><span class="k-line-number-rows">
+      >
         <span
-          v-for="line in matchingLineNumbers"
-          :key="line"
-          class="k-line"
+          v-if="!isSingleLine"
+          class="k-line-number-rows"
         >
-          <a
-            :id="`${linePrefix}-L${line}`"
-            class="k-line-anchor"
-            :href="props.showLineNumberLinks ? `#${linePrefix}-L${line}` : undefined"
-          >{{ line }}</a>
+          <span
+            v-for="line in matchingLineNumbers"
+            :key="line"
+            class="k-line"
+          >
+            <a
+              :id="`${linePrefix}-L${line}`"
+              class="k-line-anchor"
+              :href="props.showLineNumberLinks ? `#${linePrefix}-L${line}` : undefined"
+            >{{ line }}</a>
+          </span>
         </span>
-      </span><code v-html="filteredCode" /></pre>
+        <code v-html="filteredCode" />
+      </pre>
 
       <pre
         v-else
         class="k-highlighted-code-block"
+        :class="{
+          'is-single-line': isSingleLine,
+          'show-copy-button': showCopyButton
```

Comment:
nit/suggestion: if this class is only responsible for truncating the code, should the class be something like `truncate-single-line-code` instead so it's more clear?